templates/thank-you.html

```diff
@@ -4,35 +4,33 @@
 {% set page_title = _("We’ve received your answers") + " - " + page_header.title %}
 
 {% block main %}
-  {% if (schema_theme == "default") or (schema_theme == "northernireland") %}
-      <h1 data-qa="submission-successful-title">{{ _("Submission successful") }}</h1>
-      {% call onsPanel({
-            "spacious": true,
-            "type": "success",
-            "classes": "u-mb-s"
-          }) %}
-        <p>
-          {% if metadata.trad_as %}
-              {{ _("Your answers were submitted for <span>{ru_name}</span> ({trading_as_name}) on {submitted_date_time}").format(
-              ru_name = metadata.ru_name,
-              trading_as_name = metadata.trad_as,
-              submitted_date_time=metadata.submitted_time|format_datetime) }}
-          {% else %}
-              {{ _("Your answers were submitted for <span>{ru_name}</span> on {submitted_date_time}").format(
-              ru_name = metadata.ru_name,
-              submitted_date_time=metadata.submitted_time|format_datetime) }}
-          {% endif %}
-        </p>
-        <p>{{ _("Transaction ID: <b>{transaction_id}</b>").format(transaction_id=metadata.tx_id)}}</p>
-      {% endcall %}
 
-      <p class="u-mb-s">{{ _("Your answers will be processed in the next few weeks.") }}
-      {{ _("We may contact you to query your answers via phone or secure message.") }}</p>
+  <h1 data-qa="title">{{ _("Submission successful") }}</h1>
 
-      <p class="u-mb-s">{{ _("For more information on how we use this data.") }}<br><a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
-  {% else %}
-      <h1 data-qa="submission-successful-title"><span class="icon--check-green"></span>{{ _("Thank you for submitting your census") }}</h1>
+  {% call onsPanel({
+    "spacious": true,
+    "type": "success",
+    "classes": "u-mb-s"
+    })
+  %}
+    <p data-qa="guidance">
+      {% if content.trad_as %}
+        {{ _("Your answers were submitted for <span>{ru_name}</span> ({trading_as_name}) on {submitted_date_time}").format(
+          ru_name = content.ru_name,
+          trading_as_name = content.trad_as,
+          submitted_date_time=content.submitted_time|format_datetime) }}
+      {% else %}
+        {{ _("Your answers were submitted for <span>{ru_name}</span> on {submitted_date_time}").format(
+          ru_name = content.ru_name,
+          submitted_date_time=content.submitted_time|format_datetime) }}
+      {% endif %}
+    </p>
```

Comment:
This logic should be handled in the context handler and the resulting string passed to the template